src/css/override_vars.scss

```diff
@@ -1,122 +1,116 @@
 // color
-$color-blue:              #056dd4;
-$color-primary:              $color-blue;
-$color-primary-darker:       #2c3e50;
-$color-primary-darkest:      #1f2c38;
 
-$color-primary-alt:          #3498db;
-$color-primary-alt-dark:     darken($color-primary-alt, 40%);
-$color-primary-alt-darkest:  darken($color-primary-alt, 70%);
-$color-primary-alt-light:    #77b6ff;
-$color-primary-alt-lightest: #d9f1ff;
-
-$color-red:            #cb4f03;
-$color-secondary:            $color-red;
-$color-secondary-dark:       #b0392e;
-$color-secondary-darkest:    darken($color-secondary, 75%);
-$color-secondary-light:      #f3736e;
-$color-secondary-lightest:   #deeeef; // DEEF!
-
-$color-warning:              #cb4f03;
-$color-error:                #cb4f03;
-
-$color-visited:              $color-blue;
-
-$color-gold:                 #f4b400;
-$color-focus:                $color-gold;
-
-$color-green:               #0daf00;
-$color-finish:               $color-green;
-
-$color-darkgray: #555;
-$color-mediumgray: #8e8e8e;
-$color-lightgray: #d3d3d3;
-$color-lightestgray: #f5f5f3;
-$color-textblack: #333;
-$color-trueblack: #000;
-
-$color-gray-dark:       $color-darkgray; // redundant to prevent breaking
-$color-gray:             $color-mediumgray;
+// named colors
+$color-blue:                    #056dd4;
```

Comment:
All the named colors here are colors that I've included in my Sketch file as potential color options. I think, for consistency and clarity, I'd prefer to use named colors — both in the css variables, and in the sketch file. Personally, while I see the potential mild utility of generic semi-semantic color names, it's also a layer of confusion — and it's sometimes hard for me to remember my intention with `primary`, `secondary`, etc.